Makefile

```diff
@@ -51,6 +51,11 @@ migrate-arbitrary:
 cq:
 	${RUFF_COMMAND} check . && ${RUFF_COMMAND} format . && ${MYPY_COMMAND} .
 
```

Comment:
With this new command, we should add `--check` to format, so `make cq` will only check and `make cqf` will fix

Reply:
Ok I can do that.